templates/js-cypress/.actor/actor.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Broken formatting here.

Reply:
hmmm, true, eslint is apparently not working 🤔